src/applications/disability-benefits/526EZ/tests/helpers.unit.spec.jsx

```diff
@@ -183,7 +229,7 @@ describe('526 helpers', () => {
     it('should return original data if disabilities is not an array', () => {
       const clonedData = _.cloneDeep(initialData);
       const pages = [];
-      const formData = _.set(clonedData, 'disabilities', { someProperty: 'value' });
+      const formData = _.set('disabilities', clonedData, { someProperty: 'value' });
```

Comment:
This is using the normal lodash [`_.set()`](https://lodash.com/docs/4.17.10#set) (not `fp`), so I think we want the object first, then the path.

Reply:
😬